### PR TITLE
Camino brand

### PR DESCRIPTION
**Pull request by mo-c4t**
change Avalanche to Camino
change Fuji to Columbus
update chainID
change ava-labs to chain4travel

one file still contains something from avalanchego (service/rosetta.go) when I change avalanchego repo to caminogo in the import block, it cannot find the constant version.Current, although it is defined in https://github.com/chain4travel/caminogo/blob/dev/version/constants.go

### CODE REVIEW

**Comment by Chriserus** [comment]
In `mainnet/config.json`, `testnet/config.json` and `config_erc20.json` there is still Avalanche mentioned as a blockchain, shouldn't we change that as well?

File: Dockerfile [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
-# Build avalanche rosetta
+# Build camino rosetta
 # ------------------------------------------------------------------------------
 FROM golang:1.19.1 AS rosetta
 
 ARG ROSETTA_VERSION
 
-RUN git clone https://github.com/ava-labs/avalanche-rosetta.git \
-  /go/src/github.com/ava-labs/avalanche-rosetta
+RUN git clone ttps://github.com/chain4travel/camino-rosetta.git \
```
Comment (Chriserus) [inline Dockerfile]: ttps? :D

File: Makefile [hunk 2]
```diff
@@ -3,15 +3,15 @@
...
 GIT_COMMIT          ?= $(shell git rev-parse HEAD)
 GO_VERSION          ?= $(shell go version | awk {'print $$3'})
 WORKDIR             ?= $(shell pwd)
-DOCKER_ORG          ?= avaplatform
+DOCKER_ORG          ?= c4tplatform
 DOCKER_IMAGE        ?= ${DOCKER_ORG}/${PROJECT}
 DOCKER_LABEL        ?= latest
 DOCKER_TAG          ?= ${DOCKER_IMAGE}:${DOCKER_LABEL}
-AVALANCHE_VERSION   ?= v1.9.3
+CAMINO_VERSION   ?= v0.2.0
```
Comment (Chriserus) [inline Makefile]: Can you please align formatting here so it's the same as a line above?

**Review by Chriserus — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (mo-c4t) [reply Dockerfile]: fixed

File: Makefile (hunk 2 above)
Comment (mo-c4t) [reply Makefile]: fixed

**Review by Chriserus — APPROVED** [review]
LGTM

File: rosetta-cli-conf/testnet/config.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "network": {
-    "blockchain": "Avalanche",
+    "blockchain": "Camino",
     "network": "Fuji"
```
Comment (havan) [inline rosetta-cli-conf/testnet/config.json]: Should we also change the `network` here?

File: rosetta-cli-conf/testnet/config_erc20.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "network": {
-    "blockchain": "Avalanche",
+    "blockchain": "Camino",
     "network": "Fuji"
```
Comment (havan) [inline rosetta-cli-conf/testnet/config_erc20.json]: and also the `network` here?

**Review by havan — COMMENTED** [review]
I've also tested building and it builds ok.

File: rosetta-cli-conf/testnet/config.json (hunk 3 above)
Comment (mo-c4t) [reply rosetta-cli-conf/testnet/config.json]: fixed

File: rosetta-cli-conf/testnet/config_erc20.json (hunk 4 above)
Comment (mo-c4t) [reply rosetta-cli-conf/testnet/config_erc20.json]: fixed

Comment (mo-c4t) [reply rosetta-cli-conf/testnet/config_erc20.json]: tbh I don't know if it should be columbus or Columbus or it doesn't matter

**Pull request merged**